sql/catalyst/src/test/scala/org/apache/spark/sql/connector/InMemoryTable.scala

```diff
@@ -114,18 +115,21 @@ class InMemoryTable(
           case (micros: Long, TimestampType) =>
             val localDate = DateTimeUtils.microsToInstant(micros).atZone(UTC).toLocalDate
             ChronoUnit.MONTHS.between(EPOCH_LOCAL_DATE, localDate)
+          case _ => throw new IllegalArgumentException("Match: unsupported argument(s) type")
```

Comment:
In these cases, why not put the value of the thing that wasn't matched in the error?

Reply:
Yeah, that's right.